honeycombio/terraform-aws-integrations
# Include docs generation config and USAGE docs for submodules

## Short description of the changes

* Adds config that tells `make generate-docs` to generate or update terraform docs for all modules in the repo
* Adds the first version of our USAGE docs for all modules

## How to verify that this has the expected result

Run `make generate-docs` locally to see USAGE.md updates in all modules as expected (no changes or reflecting any test change)
